libs/platform/settings.cpp

```diff
@@ -42,6 +42,12 @@ string ToString<string>(string const & str)
   return str;
 }
 
+template <>
+string ToString<std::string_view>(std::string_view const & str)
```

Comment:
This is faster than reference.
```suggestion
std::string ToString<std::string_view>(std::string_view str)
```